src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -733,3 +733,18 @@ let NamespaceConvention6() =
         ))
 
     Assert.That(NotFollowingNamespaceConvention fileInfo, Is.EqualTo false)
+
+
+[<Test>]
+let NamespaceConvention7() =
+    let fileInfo =
+        (FileInfo(
```

Comment:
extra paren

Reply:
@Mersho if this has been addressed, please click on the button "Resolve conversation"